index.html

```diff
@@ -2495,6 +2571,15 @@
             <code>"NotSupportedError"</code> and abort these steps.
           </li>
           <li>
+            If the <var>options</var>.url is not an empty string and it is
+            not a valid <a>URL</a>, throw <code>"SyntaxError"</code> exception
+            and abort these steps. In addition, UA may check if
```

Comment:
Strictly speaking, the spec just says a secure context is required, which _implies_ `https`. It's up to the implementation whether for a missing scheme in the URL would _assume_ (autocomplete) `https://` or err out. Maybe being strict is the better option (but not requiring a scheme in the URL, and assuming `https://` would save some bytes in the URL. Anyway it would be correct to just say MUST instead of MAY.